stars/intel.py

```diff
@@ -47,6 +47,15 @@ def add_report(self, reference, date, report):
         # Special handling for locations to remove relative and reduce memory size
         if 'location' in report:
             report['location_root'] = report['location'].root_location.xyz
+            if report['location_root'] != (0.0, 0.0, 0.0) and hasattr(self, 'location_root') and self.location_root == (0.0, 0.0, 0.0):
+                print('\nIntel fixing location')
+            if float(date) % 1.0 == 0.99 and report['location_root'] == (0.0, 0.0, 0.0) and hasattr(self, 'location_root') and self.location_root != (0.0, 0.0, 0.0):
+                print('\nIntel adding to (0, 0, 0):', report)
+                print(' *  location:', reference.location.__dict__)
+            if float(date) % 1.0 == 0.99:
+                if report['location_root'] == (0.0, 0.0, 0.0):
+                    raise KeyError('setting location root to galactic 0')
+                raise LookupError('Reseting')
```

Comment:
Is there handling for these errors or is there a way to a way to default values instead of erroring